exercises/pangram/canonical-data.json

```diff
@@ -49,6 +49,11 @@
          "description": "Panagram in alphabet other than ASCII",
          "input": "Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.",
          "expected": false
+      },
+      {
+        "description": "test that count is case-insensitive",
```

Comment:
This needs a better description of what we're testing for.
case insensitivity is already covered by: "Five quacking Zephyrs jolt my wax bed."